src/lib/run.ts

```diff
@@ -0,0 +1,53 @@
+// Originally decaffeinated from:
+// https://github.com/harlantwood/lightsaber/blob/64a824c34f6b1bc7e745f62526fcafb103837368/src/shell.coffee
+
+import chalk from 'chalk'
+
+import lodash from 'lodash'
+const { clone, defaults, merge, isEmpty } = lodash
+
+import shelljs from 'shelljs'
+const { exec, exit } = shelljs
+
+type Options = {
+  color?: boolean
+  relaxed?: boolean
+  quiet?: boolean
+  quietCommand?: boolean
+  quietResponse?: boolean
+}
+
+export function run(command: string, options: Options = {}): string {
```

Comment:
this has options we don't currently use here, by virtue of being ported from another lib (see comment at top of file)